frontend_layout/css/style.css

```diff
@@ -0,0 +1,331 @@
+body {
```

Comment:
In my opinion, you should name *.css files more precisely because "css" is already about "style". If this css-file are built for all pages you can name it like "base.css" or something like this.

Reply:
@avplatonov, i agree with the observation. Edits made.